multicast/recv.py

```
@@ -243,7 +288,10 @@
 ])
 
 
-_w_non_multicast = f"{_w_prefix}\nJust use socket.Socket.bind(...) for non-multicast networking."
+_w_non_multicast: str = "\n".join([
+	_w_prefix,
+	"Just use socket.socket.bind(...) for non-multicast networking.",
+])
```

Ruff (0.15.21)

[warning] Consider f-string instead of string join

Replace with f-string

(FLY002)